indico/web/client/js/react/components/Datepicker.jsx

```diff
@@ -0,0 +1,78 @@
+// This file is part of Indico.
```

Comment:
When I type a date in a different format than the one specified in the regform's field settings, it accepts it (nice!) but it keep showing it in the wrong format. There is no feedback whatsoever. Could we automatically convert the text input to the proper format as soon as the input is valid?

AFAICT the flexibility only affects the separator (`/` vs `.`), but still...